### ISSUE TITLE
Question of Training

### ISSUE THREAD

**Issue by purple7seven** [JiahuiYu/generative_inpainting#393]
Thank you for your code. I trained the model with myself dataset, involve 4000 images for training and 1000 images for testing.
First, i have trained the model for ten days with four 16G GPU, and i test the model today, the result is unsatisfactory. I have no idea if there are some wrong options in my training process. This is my loss.
     
Second, i have trained the model for 121 epochs with the 4000 images for ten days. If the model have a lot of hyperparameters resulting in the training process so slow?

**Comment by JiahuiYu** [comment]
The training is generally slow. I don’t think 4K images are enough. So please collect more images first if possible. By the way, I don’t see any figure in your question.

**Comment by purple7seven** [comment]
Oh! Sorry. Here is my loss.
![image](https://user-images.githubusercontent.com/41116984/71795123-a4146000-307f-11ea-8bc0-90542d9482a5.png)
![image](https://user-images.githubusercontent.com/41116984/71795127-a8d91400-307f-11ea-989b-2eabb1e0795b.png)
![image](https://user-images.githubusercontent.com/41116984/71795130-ac6c9b00-307f-11ea-9757-99db8ea87f4b.png)
![image](https://user-images.githubusercontent.com/41116984/71795134-b0002200-307f-11ea-9624-dbcc4e43c752.png)
![image](https://user-images.githubusercontent.com/41116984/71795139-b393a900-307f-11ea-8308-27c957dcf486.png)
And this is my test result.
![output_0](https://user-images.githubusercontent.com/41116984/71795194-fa819e80-307f-11ea-932a-983b1d1681fd.jpg)
The corresponding mask is:
![0](https://user-images.githubusercontent.com/41116984/71795202-08372400-3080-11ea-97c1-492116ccd5f3.png)
This is the ground truth.
![00000002](https://user-images.githubusercontent.com/41116984/71795280-6d8b1500-3080-11ea-9b45-453fff4c8f4f.jpg)
I think even if my dataset is small, is the result wrong?
